content/md/en/docs/reference/scale-codec.md

```diff
@@ -25,55 +25,52 @@ The following table shows how the Rust implementation of the Parity SCALE codec
-| Enumerations (tagged-unions)    | A fixed number of variants, each mutually exclusive and potentially implying a further value or series of values. Encoded as the first byte identifying the index of the variant that the value is. Any further bytes are used to encode any data that the variant implies. Thus, no more than 256 variants are supported. | `Int(42)` and `Bool(true)` where `enum IntOrBool { Int(u8), Bool(bool),}` | `0x002a` and `0x0101` |    |
-
-Footnotes:
-
-[^1]: Compact/general integers are encoded with the two least significant bits denoting the mode:
-
-* `0b00`: single-byte mode; upper six bits are the LE encoding of the value (valid only for values
-of 0-63).
-* `0b01`: two-byte mode: upper six bits and the following byte is the LE encoding of the value
-(valid only for values `64-(2**14-1)`).
-* `0b10`: four-byte mode: upper six bits and the following three bytes are the LE encoding of the
-value (valid only for values `(2**14)-(2**30-1)`).
-* `0b11`: Big-integer mode: The upper six bits are the number of bytes following, plus four. The
-value is contained, LE encoded, in the bytes following. The final (most significant) byte must be
-non-zero. Valid only for values `(2**30)-(2**536-1)`.
+| Type   | Description  | Example SCALE decoded value  | SCALE encoded value  |
+|---|---|---|---|
+| Fixed-width integers  | Basic integers are encoded using a fixed-width little-endian (LE) format. | `signed 8-bit integer 69`  | `0x45` |
+|   |   | `unsigned 16-bit integer 42`  | `0x2a00`  |
+|   |   | `unsigned 32-bit integer 16777215`  | `0xffffff00` |
+| Compact/general integers[^1] | A "compact" or general integer encoding is sufficient for encoding large integers (up to 2\*\*536) and is more efficient at encoding most values than the fixed-width version. (Though for single-byte values, the fixed-width integer is never worse.) | `unsigned integer 0` | `0x00` |
+|   |  | `unsigned integer 1`  | `0x04` |
+|   |  | `unsigned integer 42`  | `0xa8` |
+|   |  | `unsigned integer 69`  | `0x1501` |
+|   |  | `unsigned integer 65535`  | `0xfeff0300` |
+|   |  | `BigInt(100000000000000)`  | `0x0b00407a10f35a` |
+| Boolean  | Boolean values are encoded using the least significant bit of a single byte. | `false`  | `0x00`  |
+|          |   | `true`  | `0x01`  |
+| Results [^2] | Results are commonly used enumerations which indicate whether certain operations were successful or unsuccessful. | `Ok(42)`  | `0x002a`  |
+|          |   | `Err(false)`  | `0x0100`  |
+| Options [^3] | One or zero values of a particular type.   | `Some`  | `0x01` followed by the encoded value  |
+|           |          | `None`       | `0x00`   |
+|           |          | `true`       |  `0x01`  |
```

Comment:
`true` and `false` don't belong to any `Option` type. How the are relevant here?

Reply:
Thanks for catching this. Proposed fix here: https://github.com/substrate-developer-hub/substrate-docs/pull/1200. Feel free to review 🙏🏻